### ISSUE TITLE
Update Documentation Examples 

### ISSUE THREAD

**Issue by pirm-in** [Tonejs/Tone.js#438]
The documentation of `PingPongDelay` and `FeedbackDelay` use a broken example.

e.g.
```
var pingPong = new Tone.PingPongDelay("4n", 0.2).toMaster();
var drum = new Tone.DrumSynth().connect(pingPong);
drum.triggerAttackRelease("C4", "32n");
```
gives
> TypeError: Tone.DrumSynth is not a constructor

Commit 2f4d6d07cdd53ceecda92278f7cb77c89122646a renames DrumSynth to MembraneSynth, so this should be updated in the examples as well. I could open a small MR for this, but documentation is in another repo which is private i guess.

**Comment by tambien** [comment]
PR's welcome. Documentation is all in JSDoc comments in the relevant classes. 

[code here](https://github.com/Tonejs/Tone.js/blob/ed71d8141bea1daacff2f98ea2df696fee24aae8/Tone/effect/PingPongDelay.js#L20)

**Comment by tambien** [comment]
fixed by #442
